src/game.cpp

```diff
@@ -2341,6 +2341,9 @@ bool game::is_game_over()
             return false;
         }
         Messages::deactivate();
+        if( get_option<bool> ( "AUTOSAVE_ON_DEATH" ) ) {
+            save();
```

Comment:
It should be `autosave` or `quicksave` here (so there would be some UI feedback, not just a silent save).

Reply:
Done.